specification/ai/Azure.AI.Projects/agents/threads/routes.tsp

```diff
@@ -93,3 +93,21 @@ op deleteThread is Azure.Core.Foundations.Operation<
   },
   ThreadDeletionStatus
 >;
+
+/**
+ * Gets a list of threads that were previously created.
+ *
+ * @returns The requested list of threads.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-names" "mirrored API responds with a container"
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "not yet an Azure operation"
+#suppress "@azure-tools/typespec-azure-core/no-operation-id" "non-standard operations"
+#suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "not yet versioned"
```

Comment:
Are all the above suppressions needed? or is this a copy-paste from some other operator? Do you mind checking?